xarray/testing/assertions.py

```diff
@@ -162,7 +176,7 @@ def assert_identical(a: DataTree, b: DataTree, from_root: bool = True): ...
 
 
 @ensure_warnings
-def assert_identical(a, b, from_root=True):
+def assert_identical(a, b, from_root=True, check_dim_order: bool = True):
```

Comment:
I don't think it makes sense to have this arg — identical means identical :) — but happy to defer if someone does...